test/compile/mark-point.test.ts

```diff
@@ -114,7 +114,15 @@ describe('Mark: Square', function() {
   });
 
   it('should be filled by default', function() {
-    // TODO
+    const model = parseModel({
```

Comment:
Here, we want to test different models parsed from different specs. If we can extract some common functionality, we can of course do that.